hubble/cmd/observe/flows.go

```diff
@@ -529,6 +529,9 @@ func newFlowsCmdHelper(usage cmdUsage, vp *viper.Viper, ofilter *flowFilter) *co
 	filterFlags.Var(filterVar(
 		"cel-expression", ofilter,
 		"Filter flows using the given CEL expression"))
+	filterFlags.Var(filterVar(
+		"interface", ofilter,
```

Comment:
I noticed the API supports filtering by name and index, should we call this `interface-name` in case we later want to add support for `interface-index`? Also should we add filtering by index in this PR, or do you think most users won't need t?

Reply:
I think if we need it at some point the CLI could set an iface index filter when the argument looks numeric, and fallback on iface name otherwise. That way the UI feels cleaner IMHO, what do you think?

Reply:
Yeah that seems reasonable.